csharp/src/Ice/Connection.cs

```diff
@@ -688,47 +688,11 @@ async Task PerformCloseAsync()
 
         private async ValueTask InvokeAsync(IncomingRequestFrame request, Current current, long streamId)
         {
-            // Notify and set dispatch observer, if any.
-            IDispatchObserver? dispatchObserver = null;
-            ICommunicatorObserver? communicatorObserver = _communicator.Observer;
-            if (communicatorObserver != null)
-            {
-                dispatchObserver = communicatorObserver.GetDispatchObserver(current, streamId, request.Size);
-                dispatchObserver?.Attach();
-            }
+            OutgoingResponseFrame response =
```

Comment:
Should this private method be renamed DispatchAsync, and use the same parameter order as DispatchAsync on ObjectAdapter?

Reply:
Yes, fixed.